src/layout/FileUpload/DropZone/DropzoneComponent.tsx

```diff
@@ -88,10 +84,9 @@ export function DropzoneComponent({
             : undefined
         }
       >
-        {({ getRootProps, getInputProps, isDragActive, isDragReject }) => {
+        {({ getRootProps, getInputProps, isDragActive }) => {
```

Comment:
Does this mean you fixed #2012? :partying_face:

Reply:
That was not the intention but it probably did 😅 I was just confused that the box turned red (and stayed red) after trying to upload something the dropzone rejected